src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java

```diff
@@ -160,4 +160,12 @@ void should_auto_complete_a_task() {
         assertNotNull(response);
         assertEquals(HttpStatus.OK, response.getStatusCode());
     }
+
+    @Test
+    void should_cancel_a_task() {
```

Comment:
Can we add some exception scenarios?

Reply:
This method does not throw any exceptions, the validations are done in service layer and the tests are written to handle all exceptions.

Reply:
Just trying to point out that we should think about the failure scenarios in general.  Could verify for instance that the taskId you pass in is the same one used in cancelling the Task.

Also, If the role assignment service is down and throws an unchecked exception there could be a test around that.  Even if the test shows that no handling is required, if in future someone adds code to handle it differently the existing test would break.